packages/devtools_app/lib/html_main.dart

```diff
@@ -65,11 +65,14 @@ void main() {
         return;
       }
 
-      // Show the Q1 2020 DevTools survey if the current date falls in range.
+      // Show the quarterly DevTools survey if the current date falls in range.
       final surveyStartDate = DateTime(2020, 3, 11);
       final surveyEndDate = DateTime(2020, 4, 10);
       final now = DateTime.now();
-      if (now.isAfter(surveyStartDate) && now.isBefore(surveyEndDate)) {
+      const surveyActive = false;
```

Comment:
nit: delete this code rather than just making it never fire. you can always revert a git commit to bring it back and we likely won't do any more surveys from the dart:html version of the app.

Reply:
deleted survey logic.